drivers/soundwire/intel_init.c

```diff
@@ -420,9 +420,11 @@ EXPORT_SYMBOL_NS(sdw_intel_acpi_scan, SOUNDWIRE_INTEL_INIT);
  * sdw_intel_probe() - SoundWire Intel probe routine
  * @res: resource data
  *
- * This creates SoundWire Master and Slave devices below the controller.
+ * This registers a platform device for each controller, and SoundWire
```

Comment:
no. this registers a platform device for each Master handled by the Controller.

Reply:
@plbossart Updated. Thanks.